packages/create-turbo/__tests__/cli.test.ts

```diff
@@ -64,170 +57,175 @@ describe("create-turbo cli", () => {
-    cleanupTestDir();
-  });
-
   describe("--no-install", () => {
-    it("default: guides the user through the process", async () => {
-      configurePackageManager(PACKAGE_MANAGERS["npm"]);
-      const cli = spawn("node", [createTurbo, "--no-install"], {});
-
-      const messages = await runInteractiveCLI(cli);
-
-      expect(messages[0]).toEqual(
-        ">>> Welcome to Turborepo! Let's get you set up with a new codebase."
-      );
+    it.concurrent.each(Object.values(PACKAGE_MANAGERS).flat())(
+      `--use-$command: guides the user through the process ($name)`,
+      async (packageManager) => {
+        configurePackageManager(packageManager);
+        let testDir = `my-${packageManager.name}-no-install-turborepo`;
+        const cli = spawn(
+          "node",
+          [createTurbo, "--no-install", `--use-${packageManager.command}`],
+          { cwd }
+        );
 
-      expect(messages[1]).toEqual(
-        `? Where would you like to create your turborepo? (./${DEFAULT_APP_NAME})`
-      );
+        const stdout = await runInteractiveCLI(cli, testDir);
 
-      expect(getPromptChoices(messages[2])).toEqual(["npm", "pnpm", "yarn"]);
+        expect(stdout).toContain(
+          ">>> Welcome to Turborepo! Let's get you set up with a new codebase."
+        );
 
-      expect(messages[3]).toMatch(
-        /^>>> Bootstrapped a new turborepo with the following:/
-      );
+        expect(stdout).toContain(
+          `? Where would you like to create your turborepo? (./my-turborepo)`
+        );
 
-      expect(
-        messages.find((msg) =>
-          msg.match(
-            new RegExp(
-              `>>> Success! Created a new Turborepo at "${DEFAULT_APP_NAME}"`
-            )
-          )
-        )
-      ).toBeTruthy();
+        expect(stdout).toContain(
+          ">>> Created a new turborepo with the following:"
+        );
 
-      expect(getGeneratedPackageJSON().packageManager).toMatch(/^npm/);
+        expect(stdout).toContain(
+          `>>> Success! Created a new Turborepo at "${testDir}"`
+        );
 
-      expect(fs.existsSync(path.join(testDir, "node_modules"))).toBe(false);
-    });
+        expect(getGeneratedPackageJSON(testDir).packageManager).toMatch(
+          new RegExp(`^${packageManager.command}`)
+        );
 
-    Object.values(PACKAGE_MANAGERS).forEach((packageManager) => {
-      it(`--use-${packageManager.command}: guides the user through the process (${packageManager.name})`, async () => {
+        expect(fs.existsSync(path.join(cwd, testDir, "node_modules"))).toBe(
+          false
+        );
+      },
+      1000 * 30
+    );
+
+    it(
+      "interactively configure",
+      async () => {
+        let packageManager = PACKAGE_MANAGERS["npm"][0];
         configurePackageManager(packageManager);
-        const cli = spawn(
+        let testDir = `my-turborepo`;
+
+        // GitHub Windows CI seems to fail to handle the first process spawned.
```

Comment:
```suggestion
turnOffJanetJackson()
```

https://devblogs.microsoft.com/oldnewthing/20220816-00/?p=106994